src/language-html/parser-html.js

```diff
@@ -340,7 +340,7 @@ module.exports = {
       recognizeSelfClosing: true,
       isTagNameCaseSensitive: true,
       getTagContentType: (tagname) => {
-        if (tagname !== "template") {
+        if (tagname !== "template" && tagname !== "html") {
```

Comment:
I'm not sure about `template`, but I'm sure about `html`, it can be uppercase :)

Reply:
stable prettier does not support it...([Playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAhAEQPIGEAqAmgAoCiABABYwC2ANgHyoAS+AsgDIMA6U55qAEYQAJgE8effgJEBLAG7l5AWlkAzALzcQaiBHIbD08gHJdEE9sm9UAejnzJx1AGcwAJ1kAHGJKhwAd3IANQBXOAAKYDg6JFMAYgBDLy8TAF8ASl5nWzdPHycBW2FxSTtWTkkQABoQCB9ZaBdkUET3dwgA4jaEZpREugDEsWbawXdEsABrOBgAZS9J2SgAc2QYd3Da6noAdUpZeBdFsDg53sOFQ7FkcBdRkGWXOHcYYgmVmkTkNQHn2oAVi4AB4AIQm01mc0SNDgHGWcB+fzggJBc2WKzocAAiqEIPAkXR-iBFu5nu5bvJwjUSZ5YLtZCIYJRkAAOAAMtS8HWeuwmXlu3Lg5PkiNqAEc8fB3vU+iBEi5lP44CIVTT3HBJbINe9Ep9vkhfkSUSBnjRZOtNiaXBisbj8YjDcjajBEoIGUyWUgAEwuiayOgY3AQGhfW7CgCsNNCz3wbr6RuJVLgAEkoKrYHMPN4YABBdNzGBiLGE55pNJAA)). We should open new issue.
```
parser=vue
```
**Input:**
```html
<!DOCTYPE html><HTML>
  <body>
    <div v-if="foo ===    'foo'">

</div>
    <script>
new Vue({el: '#app'})
    </script>
  </body>
</HTML>
```
**Output:**
```html
<!DOCTYPE html
>><HTML>
  <body>
    <div v-if="foo ===    'foo'">

</div>
    <script>
new Vue({el: '#app'})
    </script>
  </body>
</HTML>
```

Reply:
We should.

Reply:
I don't use angular, angular does not support it too

[Playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAhAEQPIGEAqAmgAoCiABABYwC2ANgHyoAS+AsgDIMA6U55qAEYQAJgE8efAQHph4yammtOkkABoQEAA4wAltADOyUAEMATmYgB3YuYRGUJulZNijGwWZNgA1nBgAylreulAA5sgwZgCucBrU9ADqlLrwBsFgcAH2qboAbqliyOAG7iChBnBmMMReYTQmyABmTpUaAFYGAB4AQl6+-gEmNHAcoXDNrXEgnV0BoWF0cACK0RDwk3RtIMFmlWbFJuHRdObqO2ahMIm6IjCUyAAcAAwaWpaViV5axe9w+3kJhoAI5reC1bQOEAmAwAWigcDgIiR5zMcFBujRtRM9UaSBaW2mlRoukiMSJCyWq3WE3xUw0MBMghudweSAATAyvLo6AtcBAaA1iv8AKznaKVfBMhwE7Z5WIASSgyNgATAlx0AEFlQEYGIlptKgBfI1AA)

Reply:
#8162